packages/fuel-indexer-tests/contracts/fuel-indexer-test/out/debug/fuel-indexer-test-abi.json

```diff
@@ -262,6 +262,18 @@
     },
     {
       "typeId": 21,
+      "type": "struct AssetId",
```

Comment:
@segfault-magnet Here's the `AssetId` added to the ABI JSON. I planned to just skip these when pulling these types into the indexer (to avoid the name collision) but that doesn't work (due to my comment below)